ir/base.def

```diff
@@ -141,6 +150,8 @@ abstract Operation : Expression {
     toString{ return getStringOp(); }
 }
 
+/// A Path represents a fully qualified name for a P4 object
+/// global (absolute) names are prefixed by a '.'
```

Comment:
Today paths can only consist of a dot; we had a design where paths could be more complicated, but it is not currently used.